pkg/networkservice/common/updatepath/updatepath_test.go

```diff
@@ -41,7 +41,7 @@ func TestChain(t *testing.T) {
 	want := &networkservice.Connection{
 		Id: "conn-2",
 		Path: &networkservice.Path{
-			Index: 0,
+			Index: 2,
```

Comment:
@edwarnicke I found that in a master version by some reason index did not increment. Can you please verify this line change?

Reply:
As I can see here is used to update servers and looks like here should be 2. Looks like this patch also fixed issue with this test :)